services/QuillLMS/client/app/bundles/Shared/styles/input.scss

```diff
@@ -56,10 +56,10 @@
   &.dropdown-container:focus {
     outline: none;
     label {
-      color: #009a80 !important;
+      color: $quill-green !important;
     }
     .dropdown__control {
-      border: 2px #009a80 !important;
+      border: 1px $quill-green !important;
       border-style: none none solid !important;
     }
   }
```

Comment:
Just confirming that we wanted to make this change to the dropdown here?

Reply:
ahh yeah, this should still be 1px-- good catch